source/_integrations/velbus.markdown

```diff
@@ -31,18 +31,33 @@ ha_integration_type: hub
 
 The **Velbus** {% term integration %} is used to control [Velbus](https://www.velbus.eu/?lang=en) modules. It supports the Velbus USB, Velbus serial and a TCP/IP gateway.
 
+<div class='note'>
 The pushbutton LEDs of input modules are disabled by default. These can be enabled from the `Devices` panel in the `Configuration` page of the web interface.
+</div>
 
 {% include integrations/config_flow.md %}
 
-The port string used in the user interface or the configuration file can have 2 formats:
+### Configuration parameters
 
-- For a serial device: `/dev/ttyUSB00`
-- For a TCP/IP device: `127.0.0.1:3678`
-- For the VMBSIG module: `tls://192.168.1.9:27015`
+The port string used in the user interface or the configuration file can have different formats depending on the type of connection:
 
-## Actions
 
+- For a serial or USB devices: `/dev/ttyUSB00`
+- For a TCP/IP devices: `127.0.0.1:3678`
+- For Signum devices without authentication: `tls://192.168.1.9:27015`
+- For Signum devices with authentication: `tls://password@192.168.1.9:27015`
+
+### Removing the integration
+
+The Velbus integration and its entities can be removed by following these steps:
+
+1. Navigate to Settings > Devices & Services
+2. Find the Velbus integration and click on it
+3. Click the delete button to remove the Velbus config entry
```

Comment:
We have a snippet for this that should be included. See for example, the Twente Milieu documentation.